x/marker/legacy/v042/store.go

```diff
@@ -33,7 +35,11 @@ func MigrateMarkerPermissions(ctx sdk.Context, k MarkerKeeperI) error {
 			invalid := marker.AddressListForPermission(types.Access_Transfer)
 			// invalid permission grants exist, remediation required
 			if len(invalid) > 0 {
-				m := marker.(*types.MarkerAccount)
+				m, ok := marker.(*types.MarkerAccount)
+				if !ok {
+					err = fmt.Errorf("unable to cast MarkerAccountI to MarkerAccount: %v", marker)
+					return true
```

Comment:
Why return true instead of return err?

Reply:
The is the return value type of the function that is being passed into the iterator.  The true means stop and we are setting the err value which is outside the iterator function.